app/src/main/java/org/jellyfin/androidtv/util/DeviceUtils.java

```diff
@@ -14,6 +28,22 @@ public static boolean isFireTvStickGen1() {
         return Build.MODEL.equals(FIRE_STICK_MODEL_GEN_1);
     }
 
+    public static boolean isFireTvStick4k() {
+        return Build.MODEL.equals(FIRE_STICK_4K_MODEL);
+    }
+
+    public static boolean has4kVideoSupport() {
+        return !Arrays.asList(
```

Comment:
I'm inverting this to default to future or unknown devices supporting 4k playback.